x/coin_test.go

```diff
@@ -359,9 +359,15 @@ func TestCoinDivide(t *testing.T) {
 		"split into two pieces with a fractional division and a leftover": {
 			total:    NewCoin(1, 0, "BTC"),
 			pieces:   3,
-			wantOne:  NewCoin(0, FracUnit/3, "BTC"),
+			wantOne:  NewCoin(0, 333333333, "BTC"),
 			wantRest: NewCoin(0, 1, "BTC"),
 		},
+		"split fractional 2 by 3 should return 3 as leftover": {
+			total:    NewCoin(0, 2, "BTC"),
+			pieces:   3,
+			wantOne:  NewCoin(0, 3, "BTC"),
+			wantRest: NewCoin(0, 0, "BTC"),
+		},
```

Comment:
I think this is not correct. 
```
2 / 3 results in 0 and "2 left"
```

`wantRest` should be 2 fractional
`wantOne` should be zero.

Reply:
Yes, the test is incorrect...